content/en/docs/contribute/generate-ref-docs/metrics-reference.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Generating reference documentation for metrics
+content_type: task
+weight: 100
+---
+
+<!-- overview -->
+
+This page demonstrates the generation of metrics reference documentation
+
+## {{% heading "prerequisites" %}}
+
+{{< include "prerequisites-ref-docs.md" >}}
+
+<!-- steps -->
+
+## Cloning the Kubernetes repository
+
+The metric generation happens in the Kubernetes repository. 
+To clone the repository, Change directories to where you want the clone to exist.
+
+Then, execute the following command:
+
+```shell
+git clone https://www.github.com/kubernetes/kubernetes 
+```
+
+This creates a `kubernetes` folder in your current working directory.
+
+## Generate the metrics
+
+Inside the cloned repository, You will find a `test/instrumentation/documentation` directory.
+This is where the metrics are generated and maintained
+
+With each release, new metrics are added and the generated documentation must be kept up to date.
+
+To generate the latest metrics, Make sure you are in the root of the cloned Kubernetes directory. 
+Then, execute the following command:
+
+```shell
+./test/instrumentation/update-documentation.sh
+```
+
+To check for changes, execute:
+
+```shell
+git status
+```
+
+The output is similar to
+
+```
+./test/instrumentation/documentation/documentation.md
+./test/instrumentation/documentation/documentation-list.yaml
+```
+
+## Copy the generated metrics over to the website repository

```

Comment:
Copy the generated metrics documentation file to the Kubernetes website repository